docker-compose.yml

```diff
@@ -50,6 +50,7 @@ services:
         image: "nginx:1-alpine"
```

Comment:
I think having a docker compose file maybe to bring up the database is a good idea. Can someone please tell me the reason why we simply can't use the node server frontend: yarn start and backend nodemon server.js. Adding these extra files, and doing npm installs in the container seems like a bad idea.